spec/components/grid-item-spec.js

```diff
@@ -0,0 +1,81 @@
+import createComponent from '../helpers/createComponent.js';
+import GridItem, { getFormattedName } from '../../src/renderer/components/grid-item.jsx';
+import generatorsList from '../fixtures/generators-list.js';
+
+
+function mockContent() {
+  const contentDiv = window.document.createElement('div');
+  contentDiv.id = 'content';
+  contentDiv.style.height = '800px';
+  window.document.body.appendChild(contentDiv);
+}
+
+function dismissContent() {
+  window.document.body.removeChild(
+    window.document.getElementById('content')
+  );
+}
+
+describe('Grid Item', () => {
+  let renderOutput;
+
+  beforeEach(() => {
+    mockContent();
+    renderOutput = createComponent(
+      GridItem,
+      generatorsList[0]
```

Comment:
We should use a mock generator list here to ensure all tests are running always with the same values. I'm wondering why Travis is not failing, because I would expect `generatorsList` to be empty there.